ext/js/language/ja/japanese-transforms.js

```diff
@@ -822,8 +1089,16 @@ export const japaneseTransforms = {
                 suffixInflection('くあります', 'い', ['v'], ['adj-i']),
             ],
         },
-        'potential': {
-            name: 'potential',
+        'potential form': {
```

Comment:
![image](https://github.com/themoeway/yomitan/assets/24891609/4fae46f1-6a76-4a53-bc60-4c6a731a187a)
I'd remove the word 'form' from all the names to reduce clutter. The names should be brief, even if inexact